packages/cli/package.json

```
@@ -72,7 +72,7 @@
     "@prisma/generator-helper": "workspace:*",
     "@prisma/get-platform": "3.16.0-14.4908cf1dde73491641a0a47fe5f3bebd2430350c",
     "@prisma/migrate": "workspace:*",
-    "@prisma/sdk": "workspace:*",
+    "@prisma/internals": "workspace:*",
     "@prisma/studio": "0.463.0",
     "@prisma/studio-server": "0.463.0",
```

```suggestion
    "@prisma/studio": "0.464.0",
    "@prisma/studio-server": "0.464.0",
```

The latest [version](https://www.npmjs.com/package/@prisma/studio) of Studio has been released.